src/components/Tag/Tag.scss

```diff
@@ -96,4 +98,46 @@ $icon-size: rem(16px);
     cursor: default;
     pointer-events: none;
   }
+
+  &.segmented {
+    margin-left: -1 * spacing(extra-tight);
+
+    &::after {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+  }
+}
+
+.Link {
+  display: inline-grid;
```

Comment:
This seems like it should be a flex layout to me. Curious to know more about this choice as well though

Reply:
Ah strange, well if this works it's better than flex hacks for sure 👍